packages/core/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -22,9 +22,11 @@ import { Link as RouterLink } from '../Link';
 export type IconLinkVerticalProps = {
   icon?: React.ReactNode;
   href?: string;
+  onClick?: React.MouseEventHandler<HTMLAnchorElement>;
   disabled?: boolean;
   label: string;
-  action?: React.ReactNode;
+  testId?: string;
```

Comment:
My reasoning was that text is more volatile than a test id, but perhaps we should rename this to `data-testid`, for consistency sake. I'm not aware of what `getByTestId().nearest()` would do (using enzyme in the last few), but if it's about finding element positionally I find this problematic since tests would fail if elements are added/removed/rearranged. 

...

Reading a bit on `closest`... Would be try and find the closest link? Can't do as this is basically a list of links. If we're to find the closest by text, same problem above.

WDYT?

Reply:
https://kentcdodds.com/blog/testing-implementation-details

Stuff like that - getByText, get in the mind of the user's behavior and experience ("i see a button that says X so I click that") etc.


If you can make it work like that, you could sidestep the problem entirely and leave out the prop

Reply:
Done!

Reply:
Still here tho?